mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -65,8 +65,8 @@
 _logger = logging.getLogger(__name__)
 
 
-def _require_arg_unspecified(arg_name, arg_value, default_value=None, message=None):
-    if arg_value != default_value:
+def _require_arg_unspecified(arg_name, arg_value, default_values=[None], message=None):
```

Comment:
```suggestion
def _require_arg_unspecified(arg_name, arg_value, default_values=None, message=None):
    default_values=[None] if default_values is None else default_values
```

Using a mutable object as a default value is an anti-pattern.

```
```